extensions/objects/models/objects_test.py

```diff
@@ -389,6 +389,30 @@ def test_set_of_html_string(self):
         self.check_normalization(
             objects.SetOfHtmlString, mappings, invalid_values)
 
+    def test_fraction(self):
+        """Tests objects of type Fraction"""
+        mappings = [(self._create_fraction_dict(True, 0, 0, 0),
+                     self._create_fraction_dict(True, 0, 0, 0),
```

Comment:
Should have closing paren at end?

Also, consider writing like this:

        mappings = [(
            self._create_fraction_dict(True, 0, 0, 0),
            self._create_fraction_dict(True, 0, 0, 0)
        ), (
            ...,
            ...
        )]

Reply:
Done.